default/empty-project/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Made with Thimble</title>
+    <title>THIMBLE_SAMPLE</title>
```

Comment:
Let's make this more "human readable" vs. looking like code: can you make it be `Thimble Sample` (no underscore, title case)?